charts/jaeger-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.46.0
+version: 2.46.2
```

Comment:
should this be 2.46.1?

Reply:
My fork wasn't up to date: https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger-operator/Chart.yaml#L4